SPECS/glusterfs.spec

```diff
@@ -1642,6 +1642,10 @@ exit 0
 %endif
 
 %changelog
+* Tue Aug 01 2023 Yann Dirson <yann.dirson@vates.tech> - 9.6-1
```

Comment:
I think that's not the date you intended :D

Reply:
fixed